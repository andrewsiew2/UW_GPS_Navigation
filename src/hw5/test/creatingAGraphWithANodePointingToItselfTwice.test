# Test if a node can point to itself twice

# Create a graph
CreateGraph graphWithANode

# Add a pair of nodes
AddNode graphWithANode NodeA

# Add an edge
AddEdge graphWithANode NodeA NodeA labelA
AddEdge graphWithANode NodeA NodeA labelB

# Print the nodes in the graph and the outgoing edges
ListNodes graphWithANode
ListChildren graphWithANode NodeA