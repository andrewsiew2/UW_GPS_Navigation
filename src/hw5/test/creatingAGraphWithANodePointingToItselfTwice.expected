# Test if a node can point to itself twice

# Create a graph
created graph graphWithANode

# Add a pair of nodes
added node NodeA to graphWithANode

# Add an edge
added edge labelA from NodeA to NodeA in graphWithANode
added edge labelB from NodeA to NodeA in graphWithANode

# Print the nodes in the graph and the outgoing edges 
graphWithANode contains: NodeA 
the children of NodeA in graphWithANode are: NodeA(labelA) NodeA(labelB)